static/templates/subscription_creation_form.handlebars

```diff
@@ -1,6 +1,7 @@
 <div class="hide" id="stream-creation" tabindex="-1" role="dialog"
     aria-label="{{t 'Stream creation' }}">
     <form id="stream_creation_form" class="form-inline">
+        <div class="alert stream_change_property_info"></div>
```

Comment:
Could you please explain in the commit description as to what's happening here and how I can test it? Thanks! :)